lib/StreamingStore.ts

```diff
@@ -40,24 +40,68 @@ implements RDF.Source<Q>, RDF.Sink<RDF.Stream<Q>, EventEmitter> {
     }
   }
 
-  protected importToListeners(stream: RDF.Stream<Q>): void {
-    stream.on('data', (quad: Q) => {
-      for (const pendingStream of this.pendingStreams.getPendingStreamsForQuad(quad)) {
-        if (!this.ended) {
-          pendingStream.push(quad);
-          pendingStream.emit('quad', quad);
-        }
+  private onReadable(stream: RDF.Stream<Q>, storeImportStream: PassThrough): void {
```

Comment:
`stream` may contain errors (e.g. when it originates from a parser), so we should look into forwarding those error events to `storeImportStream` I suspect.

Reply:
Yes, shouldn't these errors also be propagated to the `pendingStreams`?

Reply:
I think they should not, because the error will be forwarded to `innerstore.import`, which in its turn returns an `EventEmitter` that should forward the error as well. So the error will end up at the caller of `streamingstore.import`.